deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml

```diff
@@ -42,9 +42,9 @@ spec:
       name: api-port
     - port: 25333
       targetPort: python-api-port
-      { { - if and (eq .Values.api.service.type "NodePort") .Values.api.service.nodePort } }
-      nodePort: { { .Values.api.service.nodePort } }
-      { { - end } }
+      {{ - if and (eq .Values.api.service.type "NodePort") .Values.api.service.nodePort }}
```

Comment:
Sorry I missed this, the space between `{{` and `-` should be removed too

Reply:
> Sorry I missed this, the space between `{{` and `-` should be removed too

Done~